example/presto_standalone/main.tf

```diff
@@ -66,12 +68,20 @@ module "minio" {
   host                            = "127.0.0.1"
   port                            = 9000
   container_image                 = "minio/minio:latest" # todo: avoid using tag latest in future releases
+  vault_secret                    = {
+                                      use_vault_provider   = false,
+                                      vault_kv_policy_name = "",
+                                      vault_kv_path        = "",
+                                      vault_kv_access_key  = "",
+                                      vault_kv_secret_key  = ""
+                                    }
   access_key                      = "minio"
   secret_key                      = "minio123"
   buckets                         = ["default", "hive"]
   container_environment_variables = ["JUST_EXAMPLE_VAR1=some-value", "ANOTHER_EXAMPLE2=some-other-value"]
-  use_host_volume                 = true
   data_dir                        = "/minio/data"
+  use_host_volume                 = true
+  use_canary                      = false
```

Comment:
This should be `true` ?

Reply:
It's turned on for Presto, but sure why not 🤷 Done!